enable_logging_API.gs

```diff
@@ -0,0 +1,25 @@
+/**
+* Enable the logging API for cloud project
+* @param {Number} projectNumber of the GCP project
+* @return {Boolean} API is enabled or not
+*/
+function enableLogginApisPvt(projectNumber) {
+  var name = "projects/" + projectNumber + "/services/logging.googleapis.com";
+  var url = "https://serviceusage.googleapis.com/v1/" + name + ":enable";
+  var header = {
+    "Authorization" : "Bearer " + ScriptApp.getOAuthToken(),
+  };
+  var options = {
+    'method' : 'post',
+    'headers': header,
+    'muteHttpExceptions': false,
+  };
+  try {
+    var response = UrlFetchApp.fetch(url, options);
+    var json = JSON.parse(response.getContentText());
```

Comment:
Value assigned to local variable json is never read. Either use it or don't parse.
However how you are making sure that json response doesn't have any error inside it?

Reply:
Updated the json part
If the logging API is not enabled then it would throw an error otherwise the API has been enabled